### PR TITLE
Add blender-beta

### PR DESCRIPTION
**Pull request by 3van**
Now with better blending.

### CODE REVIEW

File: Casks/blender-beta.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  version '2.74rc3'
+  sha256 '475be893aaba8b6f82da3b378a69442626502d7fa69380be1d8e71d2df843a56'
+
+  url 'http://download.blender.org/release/Blender2.74/blender-2.74-rc3-OSX_10.6-x86_64.zip'
+  name 'Blender'
+  homepage 'http://www.blender.org/'
+  license :gpl
+
+  app 'Blender/Blender.app'
+  app 'Blender/BlenderPlayer.app'
```
Comment (vitorgalvao) [inline Casks/blender-beta.rb]: It’s missing an `end` after this.

**Comment by 3van** [comment]
Oops. Fixed.

**Comment by vitorgalvao** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git).

**Comment by 3van** [comment]
...oops again, I thought I pushed the squashed commit. It's late, I'm tired.

Fixed now!

**Comment by vitorgalvao** [comment]
Looks good. Thank you.

**Pull request merged**